mesheryctl/internal/cli/root/registry/generate.go

```diff
@@ -417,6 +425,16 @@ func logModelGenerationSummary(modelToCompGenerateTracker *store.GenerticThreadS
 	utils.Log.Info(fmt.Sprintf("-----------------------------\n-----------------------------\nGenerated %d models and %d components", totalAggregateModel, totalAggregateComponents))
 }
 
+func logError(err error, modelName string) {
```

Comment:
No need.
Use log instance,  refer the existing approach which logs its output to registry log file.
Create a new log instance for errors logs when an error occurs use that log instance. As you initialize that instance set its log output to registry-generate-error.

Also make the directory structured, if you would.
`logs/registry/1. log (contains logs of al severity except error logs) 2. `error (contains only error logs)`

Reply:
Will do so positively.